15:20:16.932 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeALl - 한번 실행
15:20:16.946 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:20:16.986 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:20:16.990 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:22:35.773 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeALl - 한번 실행
15:22:35.788 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:22:35.838 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:22:35.842 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:22:35.848 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:24:36.919 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeALl - 한번 실행
15:24:36.934 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:24:36.964 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:24:36.969 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:24:36.974 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:29:19.328 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeALl - 한번 실행
15:29:19.344 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:29:51.127 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeALl - 한번 실행
15:29:51.143 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:30:11.712 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeALl - 한번 실행
15:30:11.764 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:30:11.840 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:30:11.845 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:30:11.849 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
15:30:11.852 [main] INFO  com.kdt.progmrs.kdt.OrderTesterTest - @BeforeEach
18:25:51.938 [main] INFO  c.k.p.k.jdbc.JdbcCustomerRepository - customerId -> 56fe9813-04f2-337d-bc19-b8b338a9b2b9, name -> tester00
18:25:51.945 [main] INFO  c.k.p.k.jdbc.JdbcCustomerRepository - customerId -> 4dd49478-cdcc-3fbe-93c9-41ef64d79ab8, name -> tester01
18:25:51.945 [main] INFO  c.k.p.k.jdbc.JdbcCustomerRepository - customerId -> 4d57c79e-98e8-391e-8ccc-e964ad75598a, name -> tester02
18:29:23.707 [main] ERROR c.k.p.k.jdbc.JdbcCustomerRepository - Got error while closing connection
java.sql.SQLException: Column 'create_at' not found.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.result.ResultSetImpl.findColumn(ResultSetImpl.java:584)
	at com.mysql.cj.jdbc.result.ResultSetImpl.getTimestamp(ResultSetImpl.java:985)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.main(JdbcCustomerRepository.java:22)
18:30:10.282 [main] INFO  c.k.p.k.jdbc.JdbcCustomerRepository - customerId -> 56fe9813-04f2-337d-bc19-b8b338a9b2b9, name -> tester00, craeteAt -> 2022-04-11T09:03:47
18:30:10.284 [main] INFO  c.k.p.k.jdbc.JdbcCustomerRepository - customerId -> 4dd49478-cdcc-3fbe-93c9-41ef64d79ab8, name -> tester01, craeteAt -> 2022-04-11T09:03:47
18:30:10.284 [main] INFO  c.k.p.k.jdbc.JdbcCustomerRepository - customerId -> 4d57c79e-98e8-391e-8ccc-e964ad75598a, name -> tester02, craeteAt -> 2022-04-11T09:03:47
18:37:51.680 [main] INFO  c.k.p.k.jdbc.JdbcCustomerRepository - customerId -> 4dd49478-cdcc-3fbe-93c9-41ef64d79ab8, name -> tester01, craeteAt -> 2022-04-11T09:03:47
18:43:31.795 [main] ERROR c.k.p.k.jdbc.JdbcCustomerRepository - Got error while closing connection
java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?' at line 1
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1201)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.findNames(JdbcCustomerRepository.java:25)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.main(JdbcCustomerRepository.java:44)
18:44:54.010 [main] ERROR c.k.p.k.jdbc.JdbcCustomerRepository - Got error while closing connection
java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?' at line 1
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1201)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.findNames(JdbcCustomerRepository.java:24)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.main(JdbcCustomerRepository.java:42)
18:46:55.618 [main] ERROR c.k.p.k.jdbc.JdbcCustomerRepository - Got error while closing connection
java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?' at line 1
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1201)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.findNames(JdbcCustomerRepository.java:24)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.main(JdbcCustomerRepository.java:42)
18:47:44.343 [main] ERROR c.k.p.k.jdbc.JdbcCustomerRepository - Got error while closing connection
java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ':?' at line 1
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1201)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.findNames(JdbcCustomerRepository.java:24)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.main(JdbcCustomerRepository.java:42)
18:48:04.057 [main] ERROR c.k.p.k.jdbc.JdbcCustomerRepository - Got error while closing connection
java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '?' at line 1
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1201)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.findNames(JdbcCustomerRepository.java:24)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.main(JdbcCustomerRepository.java:42)
19:10:12.944 [main] ERROR c.k.p.k.jdbc.JdbcCustomerRepository - Got error while closing connection
java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'newEmail@asd.com' for key 'customers.unq_user_email'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:117)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1098)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.insertCustomer(JdbcCustomerRepository.java:73)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.main(JdbcCustomerRepository.java:91)
19:32:04.858 [main] ERROR c.k.p.k.jdbc.JdbcCustomerRepository - Got error while closing connection
java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'newEmail@asd.com' for key 'customers.unq_user_email'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:117)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1098)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.insertCustomer(JdbcCustomerRepository.java:75)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.main(JdbcCustomerRepository.java:125)
19:32:04.930 [main] ERROR c.k.p.k.jdbc.JdbcCustomerRepository - Got error while closing connection
java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'newEmail@asd.com' for key 'customers.unq_user_email'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:117)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1098)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.insertCustomer(JdbcCustomerRepository.java:75)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.main(JdbcCustomerRepository.java:126)
19:34:25.315 [main] ERROR c.k.p.k.jdbc.JdbcCustomerRepository - Got error while closing connection
java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'newEmail@asd.com' for key 'customers.unq_user_email'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:117)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1098)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.insertCustomer(JdbcCustomerRepository.java:75)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.main(JdbcCustomerRepository.java:125)
19:34:25.382 [main] ERROR c.k.p.k.jdbc.JdbcCustomerRepository - Got error while closing connection
java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'newEmail@asd.com' for key 'customers.unq_user_email'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:117)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1098)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.insertCustomer(JdbcCustomerRepository.java:75)
	at com.kdt.progmrs.kdt.jdbc.JdbcCustomerRepository.main(JdbcCustomerRepository.java:126)
